src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const newRobot = {};
+  const newKeys = Object.keys(robot).reverse();
+  const newValues = Object.values(robot).reverse();
```

Comment:
Simplify the solution Object.keys and Object.values is redundant here

Reply:
if I could )) tell me how...